app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
Probably, you can end up writing here some code moving forward. However, I'd recommend familiarizing yourself with [the following article](https://thoughtbot.com/blog/reduce-application-clutter-disable-unwanted-rails). Using the knowledge shared there, you can remove unnecessary generators, such as, for example, helpers rails generator.

Reply:
@goodniceweb nice article. Fixed